lib/types.nix

```
@@ -1162,6 +1162,13 @@ rec {
         nestedTypes.finalType = finalType;
       };
 
+    # A list of attrnames is coerced into an attrset of bools by
+    # setting the values to true.
```

Not to tempt contributors into improving this duplicate doc.

```suggestion
    /** See https://nixos.org/manual/nixos/unstable/#sec-option-types-basic */
```